cmd/livepeer/starter/starter_test.go

```diff
@@ -99,3 +102,66 @@ func TestParseGetBroadcasterPrices(t *testing.T) {
 	assert.Equal(big.NewRat(1000, 1), price1)
 	assert.Equal(big.NewRat(2000, 3), price2)
 }
+
+func TestFlag_KeystorePath_File(t *testing.T) {
```

Comment:
```suggestion
func TestParseEthKeystorePath(t *testing.T) {
```

Follow the naming convention for other tests.